documentation/src/docs/metrics.md

```diff
@@ -362,14 +301,14 @@ segmentstore.segment.write_latency_ms
 - Segment Store global and per-segment read/write metrics (counters):
 ```
 // Global counters
-segmentstore.segment.read_bytes_global.Counter
-segmentstore.segment.write_bytes_global.Counter
-segmentstore.segment.write_events_global.Counter
+segmentstore.segment.read_bytes_global
```

Comment:
When you inspect the metrics in InfluxDB, the corresponding metric has the name `pravega_segmentstore_segmentstore_segment_read_bytes_global`, and the same for the rest of the metrics.
They all have prefixes like `pravega_segmentstore`, `pravega_controller`, or `pravega` as compared to the metrics manes reported here. 
The same for CSV metrics, with the difference that CSV might be not converting dots to underscores, as InfluxDB (or Micrometer?) does.

The prefixes are very confusing to whoever has to infer the actual metrics names from whatever is listed in this file, especially because there are at least three different prefixes.

Could you please add the explanation to how the prefixes are added (or update the metrics names to the be ones that actually reported)?

Reply:
There is a specific PR following this PR to update the document, so let's address all document issues there in order to not delay this PR further.

Reply:
Sounds good.